packages/lib/factorio/server.js

```diff
@@ -857,6 +857,20 @@ class FactorioServer extends events.EventEmitter {
 		}
 	}
 
+	/**
+	 * Get Factorio binary path
+	 *
+	 * Get the path to the factorio binary depending on the platform (MacOS support)
+	 *
+	 * @return {string} path to factorio binary
+	 */
+	getBinaryPath() {
+		if (process.platform === "darwin") {
+			return this.dataPath("..", "MacOS", "factorio");
+		}
+		return this.dataPath("..", "bin", "x64", "factorio");
+	}
+
```

Comment:
Is there a rationale for placing this method here in the class, seems rather random in the middle, it's related to the dataPath so shouldn't it be after that? Also the documentation is missing a mention of it not being valid before `init()` is called.  Note that this class doesn't use the getThing setThing naming of methods, it should either be `binaryPath()`, a property getter for `binaryPath`, or set as a property during `init()`.

Reply:
I think there is no point setting an internal class variable for this, so if you prefer I can rename it to binaryPath(). Also, placement is rather random, I don't know how do you order the functions.

Reply:
The order is things that are logically related are loosely grouped together. I would put this function after `dataPath`.

PS: every other function in the whole project uses `@returns` instead of `@return`.